docker-compose.yml

```diff
@@ -148,7 +148,7 @@ services:
       KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: CONTROLLER:PLAINTEXT,INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
       # Internal listener for communication to brokers from within the
       # Docker network, external listener for accessing from Docker host.
-      KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:29092,EXTERNAL://192.168.68.104:9092
+      KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:29092,EXTERNAL://localhost:9092
```

Comment:
The broker externally advertises itself on `localhost` so that consumers running locally will connect on that address, v.s. needing to update the host ip each time it changes.